xt/lib/Pherkin/Extension/pageobject_steps/nav_steps.pl

```diff
@@ -90,6 +91,11 @@
     'overhead entry' => 'PageObject::App::Parts::Overhead',
     'system defaults' => 'PageObject::App::System::Defaults',
     'system taxes' => 'PageObject::App::System::Taxes',
+    'year-end' => ' PageObject::App::Closing',
+    'Enter Inventory' => ' PageObject::App::Parts::AdjustSetup',
+    'GS Search' => ' PageObject::App::Search::GoodsServices',
+    'RPT Balance Sheet' => ' PageObject::App::Report::Filters::BalanceSheet',
```

Comment:
"balance sheet filter" ? (so that it becomes `Then I expect to see the balance sheet filter screen`?)